### PR TITLE
dell-dock: Use accurate FuProgress steps for ec and mst

### PR DESCRIPTION
**Pull request by hugh712**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ X ] Code fix
- [ ] Feature
- [ ] Documentation

made changes based on:
https://github.com/fwupd/fwupd/wiki/Daemon-Warning:-FuProgress-steps

### CODE REVIEW

**Comment by hughsie** [comment]
But... don't you need to update the steps values to whatever the journal said? I can't believe I guessed exactly the correct values in three places.

**Comment by hugh712** [comment]
I see, let me update the values.

**Comment by hugh712** [comment]
The values on the log as below, let me update the values base on it:
raw timing data was { 0.002, 2.772, 0.002, 0.002 } -- steps were set as [ 0 100 0 0 ] but should have been [ 0 100 0 0 ] at ../plugins/dell-dock/fu-dell-dock-i2c-ec.c:992 [device-busy]
raw timing data was { 0.041, 41.681 } -- steps were set as [ 10 90 ] but should have been [ 0 100 ] at ../plugins/dell-dock/fu-dell-dock-i2c-mst.c:935 [write] <-- Panamera
raw timing data was { 5.842, 167.497 } -- steps were set as [ 15 90 ] but should have been [ 3 97 ] at ../plugins/dell-dock/fu-dell-dock-i2c-mst.c:991 [write] <-- Cayenne

File: plugins/dell-dock/fu-dell-dock-i2c-mst.c [hunk 1]
```diff
@@ -933,9 +933,8 @@ fu_dell_dock_mst_write_panamera(FuDevice *device,
 	FuProgress *progress_local;
 
 	fu_progress_set_id(progress, G_STRLOC);
-	fu_progress_add_flag(progress, FU_PROGRESS_FLAG_GUESSED);
-	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_BUSY, 10, "stop-esm");
-	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 90, NULL);
+	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_BUSY, 0, "stop-esm");
+	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 100, NULL);
```
Comment (hughsie) [inline plugins/dell-dock/fu-dell-dock-i2c-mst.c:937]: can you do 1, 99 here please -- in case it get stuck we'll know what's going on

**Review by hughsie — APPROVED** [review]

**Review by CragW — APPROVED** [review]

**Pull request merged**